src/data/structure/Tuple2.java

```diff
@@ -0,0 +1,26 @@
+package data.structure;
+
+/**
+ * Supports simple pairs of arbitrary items
+ * @author benjaminposerow
+ *
+ * @param <T>
+ * @param <U>
+ */
+public class Tuple2<T,U> {
```

Comment:
Yes, I searched for it and alas, I couldn't find it.  Was very eager to use something built in.  If I end up ever building a Tuple4, then I might have to ditch this, but for now, it is a necessary evil.